en/includes/references/conditional-auth/api-reference.md

```diff
@@ -965,25 +970,26 @@ The HTTP POST function enables sending HTTP POST requests to specified endpoints
     <table>
       <tbody>
         <tr>
-          <td><code>url</code></td>
+          <td style="white-space: nowrap;"><code>url</code></td>
           <td>The URL of the endpoint to which the HTTP POST request should be sent.</td>
         </tr>
         <tr>
-          <td><code>body</code></td>
+          <td style="white-space: nowrap;"><code>body</code></td>
           <td>HTTP request body to be included in the POST request.</td>
         </tr>
         <tr>
-          <td><code>headers</code></td>
+          <td style="white-space: nowrap;"><code>headers</code></td>
           <td>HTTP request headers to be included in the POST request (optional).</td>
         </tr>
         <tr>
-          <td><code>authConfig</code></td>
-          <td>Authentication configuration to be included in the GET request (optional).</td>
+          <td style="white-space: nowrap;"><code>authConfig</code></td>
+          <td>An object containing the necessary metadata to invoke the API. See [AuthConfig](#authconfig) for more information.</td>
```

Comment:
necessary authentication metadata

Reply:
Fixed with https://github.com/wso2/docs-is/pull/4773/commits/9acc22f1d66873e2c67cb71939862187674bece9